doc/contributing/distribution.md

```diff
@@ -0,0 +1,17 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes many dependencies, such as V8, Undici and others; and comes
+bundled with standalone tools such as `npm`. It is not a goal of the Node.js
+project to provide a level playing field for all dependencies or bundled tools.
+With limited resources, the Node.js project does not intend to bundle or
+officially support multiple dependencies or tools that serve the same purpose.
+
+## Package managers
+
+The Node.js distribution includes `npm`, and does not include any other
+package managers.
```

Comment:
```suggestion
package managers.
We bundle npm because it's the _reference implementation_ of the npm registry,
as well as historical reasons.
```

Reply:
I'm not opposed to this, but if we're also adding sections for V8 and Undici then I'll need to also explain why each of those were chosen, and I don't really know. Perhaps we can leave out the history for now and it can be added in a follow up?

Reply:
I think it would be good to document those things. I can add a paragraph about undici (mostly because it's maintained by the same people who maintain the node core http client).

Reply:
I don't think we need to leave out the history, but also we could just say something like:

> The decision was made many years before alternatives exist, and unless there is strong reason to switch away from the chosen solution the decision will not be regularly revisited.

Reply:
I think we could start by explaining those that people can add a suggestion with the explanation (as Matteo volunteered for undici) and just leave the others blank until if/when sombody PRs in an explanation.